### ISSUE TITLE
Consider moving back to 2D

### ISSUE THREAD

**Issue by alice-i-cecile** [Leafwing-Studios/Emergence#997]
# In favor of 2D

- water dynamics kind of suck
  - expensive
  - hard to understand and use for gameplay
  - dumb stability problems that are very hard to solve
- animations are harder, especially for moving up and down slopes
- occluding structures is annoying
- reproducible builds are harder
- art is substantially more expensive, and much harder to make look nice with prototype art
- 2D is more familiar to factory builder players

# In favor of 3D

- already built
- water dynamics are wildly cool to watch
- very pretty screenshots, even with very crude models
- 3D terrain generation is pretty
- cool mechanics with height available
- 2D is more familiar to factory builder players

**Comment by alice-i-cecile** [comment]
We should be able to tackle this incrementally: migrating gameplay logic first, then graphics.

>  It might be possible to have a 2d-like gameplay if limiting your heights to ground/water (2 heigths?), and get a feeling of how it's playing there, if the concept is validated you can refactor the whole thing for more efficiency 🤔

From @vrixyz.

**Comment by TimJentzsch** [comment]
So in the end the goal is to move both gameplay and graphics to 2D?

Would definitely simplify things on the logic side.
On the graphics side we have to keep in mind that it comes with it's own set of challenges, e.g. the texture layering issues in Factorio come to mind.

**Comment by kanerogers** [comment]
Yes, seconding the “Factorio problem”, though it should be possible to use
a 3D rendering pipeline but with a fixed isometric camera, giving the
appearance of a 2D game without all the pesky sprite rendering issues.

**Comment by alice-i-cecile** [comment]
Yep, I think that gameplay is the most important to figure out now. We can swap the graphics over (or leave it as 3D with a fixed perspective) once we settle on a style we're happy with.
